### DISCUSSION TITLE
Have note links be considered in the weighing of relevant notes

### DISCUSSION THREAD

**Discussion by RealRavens** [brianpetro/obsidian-smart-connections#424] [Ideas]
Just making a separate discussion for this comment thread: https://github.com/brianpetro/obsidian-smart-connections/discussions/268#discussioncomment-8185201

Will continue the discussion here.

**Comment by brianpetro** [comment]
@RealRavens I appreciate your initiative in getting these discussions started!

I imagine the discussions section to be best for people to engage with others in the community, while I see the issues section as being more focused on threads that directly impact the development/progress of Smart Connections. I know this needs to be better explained somewhere.

Because of the above, I'm also tracking issues more closely internally, so let's use this thread instead https://github.com/brianpetro/obsidian-smart-connections/issues/423

🌴

**Reply by brianpetro** [reply]
Maybe not better explained, but starting to explain in https://github.com/brianpetro/obsidian-smart-connections/discussions/425#discussion-6099045

**Comment by RealRavens** [comment]
Thanks, was about to delete the post haha 😅
